### PR TITLE
Update issue template to include request for steps to reproduce, and expected behavior

### PR DESCRIPTION
**Pull request by jaszhix**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
 Before opening an issue, please read the guidelines for contributing linked above. Issues with invalid formats, or issues not mentioning the maintainer of the applet, may not get a response, and eventually will be closed. You can find the author by visiting the applet's page on the Spices website (https://cinnamon-spices.linuxmint.com/), or going to the applet's directory on this repository, and checking the the author field in info.json. If it is set to none in info.json, or there is no author listed on the site, then this step is not necessary.
 
-[Notify author of applet]: @<GithubAuthor>
+```
+ * Notify author of applet @<GithubAuthor>
+ * Cinnamon version (cinnamon --version)
+ * Distribution - (Mint 17.2, Arch, Fedora 25, etc...)
+ * Graphics hardware *and* driver used
+ * 32 or 64 bit
+ * Applet version (Right-click the applet, then click "About...", or go to its directory and include the version/last-edited values in metadata.json.)
```
Comment (NikoKrause) [inline .github/ISSUE_TEMPLATE]: Maybe shorten to:
`Applet version/Build date (to be found in "About.." or in "metadata.json":

File: .github/ISSUE_TEMPLATE [hunk 2]
```diff
@@ -1,15 +1,27 @@
-Have you read the guidelines for contributing?
-
 Before opening an issue, please read the guidelines for contributing linked above. Issues with invalid formats, or issues not mentioning the maintainer of the applet, may not get a response, and eventually will be closed. You can find the author by visiting the applet's page on the Spices website (https://cinnamon-spices.linuxmint.com/), or going to the applet's directory on this repository, and checking the the author field in info.json. If it is set to none in info.json, or there is no author listed on the site, then this step is not necessary.
```
Comment (NikoKrause) [inline .github/ISSUE_TEMPLATE]: Shorten this:
`Add applets UUID in title.`

File: .github/ISSUE_TEMPLATE [hunk 3]
```diff
@@ -1,15 +1,27 @@
...
 
-[Notify author of applet]: @<GithubAuthor>
+```
+ * Notify author of applet @<GithubAuthor>
```
Comment (NikoKrause) [inline .github/ISSUE_TEMPLATE]: to be found here: https://github.com/linuxmint/cinnamon-spices-applets/blob/master/.github/CODEOWNERS

File: .github/ISSUE_TEMPLATE (hunk 1 above)
Comment (NikoKrause) [reply .github/ISSUE_TEMPLATE]: Also I would put this to the top, since it's more important to know, than e.g. 64 or 32 bit

**Comment by jaszhix** [comment]
Changed.

**Comment by NikoKrause** [comment]
```
@jaszhix
```

**Comment by NikoKrause** [comment]
Looks good to me. Last think I noticed. The @GitHubAuthor is in the code block. The author won't get notifed^^

**Comment by NikoKrause** [comment]
No I mean the codeblock as this one
```
@clefebvre
```

Not the brackets

**Comment by NikoKrause** [comment]
I always wanted to try, if I can add a commit. And it works :)
Hope you're ok with it.

**Comment by jaszhix** [comment]
Looks good to me!

**Pull request merged**